polaris-react/src/components/Badge/README.md

```diff
@@ -260,7 +260,9 @@ Use to indicate when a given task has been partially completed. For example, whe
 Use to indicate when a given task has been completed. For example, when merchants have fulfilled an order.
 
 ```jsx
-<Badge progress="complete">Fulfilled</Badge>
+<Badge progress="complete" status="new">
+  Fulfilled
+</Badge>
```

Comment:
> To avoid the situation above where there's a mismatch between pip hue and badge's background

Does this mean consumer's previously passing in `progress='complete'` will need to pass in `status=new` to achieve the same effect?

Reply:
To get the result bellow `status=new` is necessary. Other wise, Pip will be `status= info` which is blue
<img width="77" alt="image" src="https://user-images.githubusercontent.com/1577809/169796457-a70c675d-3a04-4bcd-b06b-71da691c316f.png">

There are 3 alternatives to render the default  `<Badge `progress='complete' />` with bkg and pip as gray:
1)  Make `status='new'` in Badge's props
2) Make `status='new'` in Pip's props
3) Remove default `status in Pip's props

Reply:
It could be 🤦 even though the component shouldn't have 2 different colors